### DISCUSSION TITLE
Device verification email is sending to an old, deactivated email

### DISCUSSION THREAD

**Discussion by BoruiZ** [community/community#120687] [New to GitHub]
I created my account by using my university's email, but now since I graduated the email is deactivated so I am using my email as the primary email (and it's verified). But when I tried to log in to my account on a different device, it still showed sending a verification code to my deactivated uni email. Can someone help me with this? Thanks in advance!

FYI 2FA is enabled but it doesn't look like the solution for verifying new devices.

**Comment by david-wei-01001** [comment]
When you have Two-Factor Authentication (2FA) enabled on an account and you're attempting to log in from a new device, some platforms will still want to verify the device in addition to the 2FA process. This often involves sending a verification code to the primary email associated with the account.

If the verification email is still being sent to your old university email which is now deactivated, here are the steps you should take:

1. **Review Primary Email Settings**:
   - Check the settings in your account to ensure that your new email is set as the primary email address. It's possible that the university email might still be set as the primary or recovery email.
   
2. **Update Recovery Information**:
   - Make sure to update not just your primary email, but also any recovery emails or phone numbers associated with your account.

3. **Contact Support**:
   - If you're unable to change the primary email because the platform is sending a verification to the old email, you'll likely need to contact customer support for the service you're trying to access.
   - Provide them with any proof of identity they might need to verify that you are the owner of the account.

4. **Documentation**:
   - Have documents ready that prove your identity and your current email. This might include your ID, proof of your graduation and university email deactivation, and proof of ownership of the new email address.

5. **Account Recovery Process**:
   - Go through the account recovery process if the platform provides one. This may involve answering security questions or providing additional information that was set up when you created the account.

6. **Future Precautions**:
   - Once you regain full control of your account, ensure that all security information is up to date, including backup codes for 2FA if available.

7. **Legal Name Change**:
   - If necessary, some platforms may require a more formal process such as submitting a request for a legal name change or email change. This process can often be started through the support or help sections of the service.

For most services, especially those that handle sensitive personal information, the customer support route is the most common way to resolve these kinds of issues. They will be able to guide you through the process of updating your account details securely. It's important to reach out to them from the contact information provided on the official website to avoid phishing attempts or scams.

**Comment by RishabhDhawad** [comment]
To resolve this, you'll likely need to update your account settings to use your new personal email as the primary contact. Since you mentioned that 2FA is enabled but doesn't seem to be the solution for verifying new devices, it's possible that there's an additional verification step or setting that needs to be adjusted.

Here's a general outline of steps you can take:

1. Log in to your account using your current credentials.
2. Navigate to your account settings or profile settings.
3. Look for an option to update your email address or primary contact information.
4. Update your email address to your current, active personal email.
5. Save the changes and check if you receive a verification email to confirm the update.
6. Once your new email address is verified, try logging in from a different device again and see if you receive the verification code to your personal email.

**Reply by BoruiZ** [reply]
Thanks @david-wei-01001 and @RishabhDhawad, I have updated my personal email as the primary contact and the only email for account verifications, but the verification code is still sending to my uni email when I try to log in on another device. And the 2FA is also not working.

**Reply by RishabhDhawad** [reply]
Your university can only help you, in my opinion, if you visit them and give them all the information. Perhaps they can get your github access and revive it.

**Comment by Racergeoffrey35** [comment]
what if you cant remember your username and password or email address you used for your account
